SCClassLibrary/Common/GUI/Base/Grid.sc

```diff
@@ -102,7 +119,7 @@ DrawGridX {
 		^commands ?? {
 			cacheKey = [range,bounds];
 			commands = [];
-			p = grid.getParams(range[0],range[1],bounds.left,bounds.right);
+			p = grid.getParams(range[0], range[1], bounds.left, bounds.right, nil, tickSpacing);
```

Comment:
I think `numTicks` should be settable here, not made to be `nil`, otherwise you effectively can't set `numTicks` in the context of `DrawGrid`.

Doing so should be pretty straightforward, [here's a basic implementation](https://github.com/supercollider/supercollider/compare/develop...mtmccrea:supercollider:topic/gridlines-set-ticks). It's not fully thought out though. A full implementation may be out of scope of this PR (what do you think?), but the important thing is not to make changes now that would prevent adding this.

Additionally I think it's good that `tickSpacing` takes precedence over `numTicks`, but not ignored if `numTicks` is not `nil`. If both are settable, and the default `tickSpacing` interferes with the behavior of `numTicks` they want, they can just set `tickSpacing = 0`.